exist-core/src/main/java/org/exist/xquery/pragmas/ProfilePragma.java

```diff
@@ -27,8 +27,8 @@
 import org.exist.xquery.value.Sequence;
 
 public class ProfilePragma extends Pragma {
-
-    public final static QName PROFILING_PRAGMA = new QName("profiling", Namespaces.EXIST_NS, "exist");
+    public final static String LOCAL_NAME = "profiling";
```

Comment:
See my comment above about why renaming this to LOCAL_NAME is problematic and needs to be reversed.

Reply:
not renamed but added